Makefile

```diff
@@ -154,6 +154,10 @@ requirements/%.txt: requirements/%.in
 	@touch -a $(subst requirements.txt,dev.txt,$@)
 	@tox -qe $(subst requirements,dev,$(basename $(notdir $@))) --run-command 'pip-compile --allow-unsafe --quiet $(args) $<'
 
+# checkformatting.txt is a symlink not an actual requirements file so it needs its own recipe.
+requirements/checkformatting.txt:
+	@ln -frs requirements/format.txt requirements/checkformatting.txt
```

Comment:
Try `rm requirements/checkformatting.txt` and then `make requirements`, it'll re-create it